doc/sphinx/Pacemaker_Explained/node-options.rst

```diff
@@ -0,0 +1,547 @@
+Per-node Configuration
+--------------------------
```

Comment:
I was on the fence, so that's fine with me.

I went with a separate section because the Nodes section is about Pacemaker's conception of nodes -- defining them, setting node attributes in the CIB, etc.

These configuration options have nothing to do with "nodes". They're options that are applied locally on the machine (node) on which they're set. In my mind the emphasis is more that these are local configs.

Reply:
Hmm, true. It should be the first chapter after the introduction, maybe "Host-local options" or "Start-up options".

Reply:
They're applied only at startup, but I don't want to give the impression that they affect only startup behavior, so I'll go with host-local.